pom.xml

```diff
@@ -264,6 +264,25 @@
           </jgit.ignore>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>enforce-maven</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireMavenVersion>
+                  <version>3.5.0</version>
```

Comment:
This allows later versions too, right?

Reply:
Since it's not a version range, the version specified is used as a lower bound.  (I run Maven 3.5.4 and there were no complaints.)